src/java/org/apache/cassandra/db/streaming/CassandraOutgoingFile.java

```diff
@@ -170,7 +170,6 @@ public void write(StreamSession session, StreamingDataOutputPlus out, int versio
             // legacy streaming is not affected by stats metadata mutation and index sumary redistribution
             CassandraStreamHeader.serializer.serialize(header, out, version);
             out.flush();
-
```

Comment:
@isaacreath this change is not necessary, please move it back so this file is not marked as changed in git.